server/module_hash.go

```diff
@@ -0,0 +1,100 @@
+package server
+
+import (
+	"encoding/hex"
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	servertypes "github.com/cosmos/cosmos-sdk/server/types"
+	"github.com/cosmos/cosmos-sdk/store/rootmulti"
+	"github.com/cosmos/cosmos-sdk/store/types"
```

Comment:
Looks like a duplicate import of `storetypes`, could we use just that one?

Reply:
I agree, this would make the diff smaller 😄

Reply:
Addressed in https://github.com/noble-assets/cosmos-sdk/pull/3/commits/c22c9af7125d181f8b0f4923b20196d8977fa880